specification/compute/resource-manager/Microsoft.Compute/2017-03-30/disk.json

```diff
@@ -674,7 +674,8 @@
         "sku": {
           "$ref": "#/definitions/DiskSku"
         }
-      }
+      },
+      "x-ms-azure-resource": true
```

Comment:
Why this is marked as `x-ms-azure-resource`? Usually `TrackedResource`, `ProxyResource`, `Resource` are marked using this tag. please remove it as it has no special meaning in other contexts.